### PR TITLE
README: core tests require quicklisp

### PR DESCRIPTION
**Pull request by cben**
Without this they all error:
`There is no package with the name QL..`
(or similar)

### CODE REVIEW

**Comment by cben** [comment]
Say, would you like to add some continuous integration running the core tests on every pull request?

GitHub Actions are probably easiest, support Linux/Mac/Windows, and the free tier should be plenty.

File: README.md [hunk 1]
```diff
@@ -110,13 +110,14 @@ Examples of running the unit tests on ECL, SBCL, and Lispworks are below:
 # I like to wrap these in rlwrap in case I need to use the REPL, but it's optional
 
 #ECL
-rlwrap ecl --load ./run-core-tests.lisp
+rlwrap ecl --load ~/quicklisp/setup.lisp --load ./run-core-tests.lisp
```
Comment (sgithens) [inline README.md:113]: On my system, the location of the ECL init for this is:

% more ~/.ecl
;;; The following lines added by ql:add-to-init-file:
#-quicklisp
(let ((quicklisp-init (merge-pathnames "quicklisp/setup.lisp" (user-homedir-pathname))))
  (when (probe-file quicklisp-init)
    (load quicklisp-init)))

File: README.md [hunk 2]
```diff
@@ -110,13 +110,14 @@ Examples of running the unit tests on ECL, SBCL, and Lispworks are below:
...
 
 #SBCL
-rlwrap sbcl --load ./run-core-tests.lisp
+rlwrap sbcl --load ~/quicklisp/setup.lisp --load ./run-core-tests.lisp
```
Comment (sgithens) [inline README.md:116]: On my system the location for the sbcl init is:

% more .sbclrc
;;; The following lines added by ql:add-to-init-file:
#-quicklisp
(let ((quicklisp-init (merge-pathnames "quicklisp/setup.lisp" (user-homedir-pathname))))
  (when (probe-file quicklisp-init)
    (load quicklisp-init)))

File: README.md [hunk 3]
```diff
@@ -110,13 +110,14 @@ Examples of running the unit tests on ECL, SBCL, and Lispworks are below:
...
 
 #Lispworks
 # From the editor run:
+(load #P"~/quicklisp/setup.lisp")
```
Comment (sgithens) [inline README.md:120]: For lispworks this goes in your init here, with the caveat of the limited personal edition, which I added some more notes too in the readme

more .lispworks

;;; The following lines added by ql:add-to-init-file:
#-quicklisp
(let ((quicklisp-init (merge-pathnames "quicklisp/setup.lisp" (user-homedir-pathname))))
  (when (probe-file quicklisp-init)
    (load quicklisp-init)))

**Comment by sgithens** [comment]
@cben Assuming you're able to run the boxer-sunrise-core tests on either ecl or sbcl locally, if you could figure out how to have these run under github actions, that'd be an amazing initial contribution ( if it's something you're interested in looking at)

**Pull request closed without merging**